Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.13-alpine
+
+RUN apk update
+RUN apk add apache2-utils bash
+RUN python -m venv /maildump
```

Comment:
I'd create and use an unprivileged users for this, no need to run maildump as root inside the container

Reply:
i.e. I cannot have a Dockerfile do useradd if I want it to run on podman. What I think I could do is to set UID=1000...

Reply:
(done)

Reply:
this is just needed for the build, isn't it? so if I publish an image to docker hub / ghcr, you should be able to run it in any case, right?

but indeed, something like `USER nobody` should be sufficient, because it does not need to write anything (besides the htpasswd file, but you can just put that in a place like /tmp or make the directory world-writable)

Reply:
I think it should be fine like this... but maybe it should be tested with docker build. Works fine with podman.